beacon-chain/p2p/rpc_topic_mappings.go

```diff
@@ -9,32 +9,74 @@ import (
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
 )
 
-// Current schema version for our rpc protocol ID.
```

Comment:
How about taking this step further? Constraint them under different versioning file names. 

`rpc_topic_mappings_V1.go`, `rpc_topic_mappings_V2.go`... etc

Reply:
Each v1, v2 would only be about 5 lines as its simply redeclaring the topics with a higher version number.

Reply:
Fair on the line numbers. The concern is more when we get it v10, how readable it becomes to have everything under single file

Reply:
I honestly hope we dont come to the stage where we have to support 10 different versions :sweat_smile: